XenAdmin/Dialogs/LicenseManager/LicenseDataGridViewRow.cs

```diff
@@ -44,7 +45,12 @@ public enum Status
             Warning,
             Information,

```

Comment:
This enum is very confusing. First of all it's the names of two of the original members: using Warning when in fact it shows a red cross icon, and Information when in fact it shows a warning icon...

Second, the docs above the two new added members Trial and Mixed make me think they rather belong to the Dialogs.LicenseStatus.HostState enum and their evaluation should happen in ShouldShowLicenseWarningText.

This enum should probably have the following 5 members:
- Ok => green tick
- Passable (or similar term) => yellow tick
- Warning => warning triangle
- Error => red cross icon
- Updating => doesn't seem we show anything here - maybe the ajax-loader?

Reply:
Updating is indeed showed while the data is loading

As for the others: you're right, I think I'm making it even more confusing this way. Will update it.

I can't think of a better option so "Passable" it is. I thought of Alert (too close to Warning), Advisory (doesn't quite match the others), Partial (also doesn't match)